### ISSUE TITLE
The search buttons tooltips are missing

### ISSUE THREAD

**Issue by cranes-bill** [QupZilla/qupzilla#2592]
The tooltips do not appear for search buttons in „Location completer“.
I tried also with Appimage and it is the same.
![tp-1](https://user-images.githubusercontent.com/5161278/35872503-ac1ed53c-0b6f-11e8-9046-7aa61ef10103.png)

**Comment by nowrep** [comment]
Yes, that's only on `xcb` as there are used special window flags for completer popup so it doesn't grab input and shortcuts work (eg. ctrl+w to close tab).
